pom.xml

```diff
@@ -13,9 +13,9 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<maven.compiler.source>${java.version}</maven.compiler.source>
 		<maven.compiler.target>${java.version}</maven.compiler.target>
-		<keycloak.version>22.0.5</keycloak.version>
+		<keycloak.version>24.0.3</keycloak.version>
```

Comment:
I'm pretty sure this is going to introduce a backwards compatibility breaking change. We still need to target Keycloak 22.

Reply:
Due to `java.version`? Changed back to 22.0.5.